README.md

```diff
@@ -1,2 +1 @@
-# sinatra_app
-Sample sinatra application
+The project allows you to authenticate via Google. Registered users can work with the database and can see the JWT before encoding and after. Also registered users can see the contents of the session on the main page.

```

Comment:
where is the link to application?
Have you deployed it after applying the changes?